packages/mux-video/examples/async-play.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
```

Comment:
ignore all of these fresh examples. They are local test cases I've set up that got unintentionally pushed by some coarse-grained adds.